### PR TITLE
Update PyPI URLs

### PR DESCRIPTION
**Pull request by dgilman**
Doc-only change. Updating the old cheese shop URLs.

### Description
I dropped the thing about cdecimal because it is only relevant to Pythons before 3.3.

### Checklist
This pull request is:

- [x] A documentation / typographical error fix
	- Good to go, no issue or tests are needed
- [ ] A short code fix
	- please include the issue number, and create an issue if none exists, which
	  must include a complete example of the issue.  one line code fixes without an
	  issue and demonstration will not be accepted.
	- Please include: `Fixes: #<issue number>` in the commit message
	- please include tests.   one line code fixes without tests will not be accepted.
- [ ] A new feature implementation
	- please include the issue number, and create an issue if none exists, which must
	  include a complete example of how the feature would look.
	- Please include: `Fixes: #<issue number>` in the commit message
	- please include tests.

**Have a nice day!**

### CODE REVIEW

File: lib/sqlalchemy/sql/sqltypes.py [hunk 1]
```diff
@@ -604,25 +604,6 @@ class Numeric(_LookupExpressionAdapter, TypeEngine):
...
-
-           import sys
-           import cdecimal
-           sys.modules["decimal"] = cdecimal
-
-       Note that the ``cdecimal`` and ``decimal`` libraries are **not
-       compatible with each other**, so patching ``cdecimal`` at the
-       global level is the only way it can be used effectively with
-       various DBAPIs that hardcode to import the ``decimal`` library.
-
```
Comment (CaselIT) [inline lib/sqlalchemy/sql/sqltypes.py:625]: we still support py2.7. was it updated there also?

Comment (dgilman) [reply lib/sqlalchemy/sql/sqltypes.py:625]: It wasn't. Even with 2.7 still supported I think it's worth dropping this now over keeping it in and forgetting to drop it later.

**Review by sqla-tester — COMMENTED** [review]
OK, this is **sqla-tester** setting up my work on behalf of **CaselIT** to try to get revision 45b77e853e09bfd64982e54ceaeb39bdffbb2dfc of this pull request into gerrit so we can run tests and reviews and stuff

**Comment by sqla-tester** [comment]
Failed to create a gerrit review, git squash against branch 'master' failed

**Comment by CaselIT** [comment]
Thanks

**Pull request merged**